specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/GetVirtualMachineInstanceView.json

```diff
@@ -64,6 +64,7 @@
           "consoleScreenshotBlobUri": "https://{myStorageAccount}.blob.core.windows.net/bootdiagnostics-myOsDisk/myOsDisk.screenshot.bmp",
           "serialConsoleLogBlobUri": "https://{myStorageAccount}.blob.core.windows.net/bootdiagnostics-myOsDisk/myOsDisk.serialconsole.log"
         },
+        "assignedHost": "/subscriptions/{subscription-Id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/myDedicatedHostGroup/myHostGroup/Hosts/myHost",
```

Comment:
Shouldn't we also show this property in the example file that shows the response for GetVirtualMachineScaleSetInstanceView.

Reply:
AssignedHost should be shown for the response to GetVirtualMachineScaleSetVMInstanceView (instead of GetVirtualMachineScaleSetInstanceView), but there is no example file for GetVirtualMachineScaleSetVMInstanceView. GetVirtualMachine has a response for a VM on PPG. Since a VM cannot be placed on both PPG and ADH at the same time, adding ADH there would make the response unrealistic.

Reply:
We should a separate example file showing GetVirtualMachineScaleSetVMInstanceView and also add one for GetVirtualMachine  which shows the response of DedicatedHost without PPG.